integration-test/Css.ts

```diff
@@ -370,4 +388,15 @@ export type Padding =
   | "paddingBottom"
   | "paddingLeft";
 
+type Brand<K, T> = K & { __brand: T };
+type Breakpoint = Brand<string, "Breakpoint">;
+
+export const sm = "@media screen and (max-width:600px)" as Breakpoint;
+export const md = "@media screen and (min-width:601px) and (max-width:960px)" as Breakpoint;
+export const smOrMd = "@media screen and (max-width:960px)" as Breakpoint;
+export const mdUp = "@media screen and (min-width:601px)" as Breakpoint;
```

Comment:
...tempted to call these `mdAndUp` / `mdAndDown` because I personally found MUI's "up(md)" to be confusing about whether md was inclusive or exclusive. I get it now, and think switching the order like this, i.e. `mdUp`, is easier to guess that "yeah its inclusive".

Reply:
`mdAndUp` makes more sense to me.

Reply:
Cool, agreed; changed to that.